src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import Exception.DeadlineFormatException;
+import Exception.TaskFormatException;
+import Exception.TodoFormatException;
+import Exception.EventFormatException;
+import Exception.TaskAlreadyDoneException;
+import Exception.DoneFormatException;
+import Exception.DeleteFormatException;
+import FileHandler.FileHandler;
+import FileHandler.FileTaskWriter;
+import TaskClass.Deadline;
+import TaskClass.Event;
+import TaskClass.TaskList;
+import TaskClass.ToDo;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static final String LOGO = " ____        _\n"
+            + "|  _ \\ _   _| | _____\n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINE_SEPERATOR = "    ____________________________________________________________";
+
+    public static void main(String[] args) throws IOException {
+        System.out.println("    Hello from\n" + LOGO);
+        System.out.println(LINE_SEPERATOR +
+                "\n    Hello! I'm Duke\n" +
+                "    What can I do for you?\n" +
+                LINE_SEPERATOR);
+        Scanner sc = new Scanner(System.in);
+        String sourceFilePath = "data" + File.separator + "task.txt";
+        File source = new File(sourceFilePath);
+        //FileHandler fileHandler;
```

Comment:
Where possible, try and remove comments that don't explain what the code is doing (including commented-out code)